src/lib/stepper/stepper.spec.ts

```diff
@@ -877,6 +867,76 @@ describe('MatStepper', () => {
     });
 
   });
+
+  describe('stepper with error state', () => {
+    let fixture: ComponentFixture<MatHorizontalStepperWithErrorsApp>;
+    let stepper: MatStepper;
+
+    beforeEach(() => {
+      fixture = createComponent(
+        MatHorizontalStepperWithErrorsApp,
+        [{
+          provide: MAT_STEPPER_GLOBAL_OPTIONS,
+          useValue: {showError: true}
+        }],
+        [MatFormFieldModule, MatInputModule]
+      );
+      fixture.detectChanges();
+      stepper = fixture.debugElement
+          .query(By.css('mat-horizontal-stepper')).componentInstance;
+    });
+
+    it('should show error state', () => {
+      let nextButtonNativeEl = fixture.debugElement
+          .queryAll(By.directive(MatStepperNext))[0].nativeElement;
+
+      stepper.selectedIndex = 1;
+      stepper._steps.toArray()[0].hasError = true;
```

Comment:
You can do `stepper._steps.first` instead of `stepper._steps.toArray()[0]` here.

Reply:
Awesome, thanks!